wiki/en/Running-a-Server.md

```diff
@@ -290,17 +259,17 @@ It is highly recommended to test your Server on a **Public Directory** first so
 
-If you are running a Server on your home network, people from outside will not be able to see things inside it. So if you want to operate a Jamulus Server, you need to open a hole in your router's firewall to let Jamulus Clients connect to it.
+If you are running a Server on your home network, people from outside will not be able to see things inside it. If you want to operate a Jamulus Server, you need to set up your router's firewall to let Jamulus Clients connect to the internal Server.
 
-**The default port for the current version of Jamulus is UDP (not TCP) port 22124.**
+**Note:** The default port for the current version of Jamulus is **UDP** (not TCP) port **22124**.
 
 The exact setup of port forwarding differs for every router. For help see [portforward.com](https://portforward.com).
 
-Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your Server (_but see also the note on dynamic DNS below_). You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a Client on the same machine as your Server, that would be `localhost` or `127.0.0.1`. 
+Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your Server (_but see also the note on dynamic DNS below_). You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a Client on the same machine as your Server, that would be `localhost` or `127.0.0.1`.
 
 ## Dynamic DNS and why you will probably need it
 
-Most domestic Internet connections will change their IP address after a period (hours, days or weeks). To make it easier for people to connect to you, you may therefore also want set up a dynamic DNS address. You can do this on [the machine](https://www.online-tech-tips.com/computer-tips/ddns-dynamic-dns-service/) that you are running the Jamulus Server on, or preferably your router [may support it](https://www.noip.com/support/knowledgebase/how-to-configure-ddns-in-router/).
+Most domestic Internet connections will change their IP address after a period (hours, days or weeks). To make it easier for people to connect to you, you might want set up a "dynamic DNS address". You can do this on [the machine](https://www.online-tech-tips.com/computer-tips/ddns-dynamic-dns-service/) that you are running the Jamulus Server on, or preferably your router [may support it](https://www.noip.com/support/knowledgebase/how-to-configure-ddns-in-router/).
 
 Note also that your home router may also change the IP address of the machine that you are running your Server on. In which case you may need to give that machine a static IP in the router's DHCP configuration.
 
```

Comment:
Lines 276 (new) and 282 (new) both want to be level 2 headings.

Reply:
Fixed. Maybe it should even be a deeper level.

Reply:
They came from a page describing the server UI and are literally the tab names on the UI, rather than section headings in a document.  All the headings refer literally to widgets on the UI.

However, that's not really appropriate here - this page isn't for describing the UI.  Hence it needs rewriting completely.

https://github.com/jamulussoftware/jamuluswebsite/issues/729

Reply:
> All the headings refer literally to widgets on the UI.

As is the case for the client manual.

> Hence it needs rewriting completely.

I think (yet another) complete re-write is a separate PR once this PR has determined how to make things shorter.